Add "Access Graph Path Changed" event tracking

jakule opened the pull request:
Implementation for Access Path changed event introduced here https://github.com/gravitational/teleport/pull/40807/files#diff-d7bcfa939d3b1966ee14f807fcfda96ace9b6480433e1574b3866cdaf1b46a7bR4074

jakule committed:
Add "Access Graph Path Changed" event tracking

The update introduces support for access path changes. The necessary objects, events, and methods are adjusted accordingly. Icon representation for the new event in the Audit Event List is also added.

jakule committed:
Add affected_resource_type field

jakule committed:
Update audit event messages and add resource kind

Updated the description and formatting of audit events when an access graph path is changed. Additionally, the "affected_resource_kind" field was added to provide more information about the affected resource.

jakule committed:
fmt

ryanclark commented on web/packages/teleport/src/services/audit/makeEvent.ts:
```diff
@@ -1823,6 +1823,16 @@ export const formatters: Formatters = {
+      affected_resource_name,
+      affected_resource_source,
+    }) =>
+      `${affected_resource_kind || 'Node'} [${affected_resource_name}/${affected_resource_source}] changed the access graph path`,
```
Should `the access graph path` be `an access path` or `an access graph path`?

jakule committed:
Update access path change message in audit events

The message text for changes to access paths in audit/audit events has been updated. This modification improves the clarity of the description for such events.
